inc/saivlan.h

```diff
@@ -192,57 +216,54 @@ typedef sai_status_t (*sai_get_vlan_attribute_fn)(
     _Inout_ sai_attribute_t *attr_list
     );
 
-/**
- * Routine Description:
- *    @brief Remove VLAN configuration (remove all VLANs).
- *
- * Arguments:
- *    None
- *
- * Return Values:
- *    @return SAI_STATUS_SUCCESS on success
- *            Failure status code on error
- */
-typedef sai_status_t (*sai_remove_all_vlans_fn)(
-    void
+/*
+    \brief Create VLAN Member
+    \param[out] vlan_member_id VLAN member ID
+    \param[in] attr_count number of attributes
+    \param[in] attr_list array of attributes
+    \return Success: SAI_STATUS_SUCCESS
+            Failure: failure status code on error
+*/
+typedef sai_status_t (*sai_create_vlan_member_fn)(
+    _Out_ sai_object_id_t* vlan_member_id,
+    _In_ uint32_t attr_count,
+    _In_ const sai_attribute_t *attr_list
```

Comment:
From a usage point of view it appears a bit un-optimized compared to the previous routine where one could add multiple ports to a vlan in a single call. Now does the vlan_member object represent a single vlan-port pair ? and does one have to call this routine as many times as the number of ports in a vlan ?
It would help to see some example usage of these new objects.